lib/geo/point.go

```diff
@@ -295,3 +295,13 @@ func (a *Point) Interpolate(b *Point, t float64) *Point {
 		a.Y*(1.0-t)+b.Y*t,
 	)
 }
+
+func (p *Point) TruncateFloat32() {
+	p.X = float64(float32(p.X))
+	p.Y = float64(float32(p.Y))
+}
+
+func (p *Point) TruncateDecimals() {
```

Comment:
these two seem to be doing the same thing. can you just combine them into `TruncatePrecision` and then call that everywhere

Reply:
truncate decimals is only used when exporting the final result. We still want at least the float32 precision until layout is completed and then we only need 3 decimals in the exported result.